src/App.jsx

```diff
@@ -1,7 +1,43 @@
-import data from "./data.json";
-
-console.log(data);
+import { Header } from "./components/Header.jsx";
+import {
+  AllAlbums,
+  SortAlbums,
+  SortSingles,
+} from "./components/RenderAlbum.jsx";
+import { useState } from "react";
+import "./components/header.css";
 
 export const App = () => {
-  return <div>Find me in src/app.jsx!</div>;
+  const [isSorted, setSorted] = useState(false);
```

Comment:
We are super impressed that you already used state in this weeks project. Well done!
The only thing we are worried if there would be an enormous amount of albums you would never reach the singles. Design wise you could maybe put them next to one another instead of after each other.
//Izabel & Linda